src/main/java/uk/gov/hmcts/divorce/bulkscan/transformation/PaperFormDetailsTransformer.java

```diff
@@ -0,0 +1,160 @@
+        );
+        caseData.getPaperFormDetails().setApplicant1LegalRepSigningSOT(
+            from(toBoolean(ocrDataFields.getSoleApplicantOrApplicant1OrLegalRepSignature()))
+        );
+        caseData.getPaperFormDetails().setApplicant1LegalRepPosition(ocrDataFields.getSoleApplicantOrApplicant1LegalRepPosition());
+
+        String day = ocrDataFields.getStatementOfTruthDateDay();
+        String month = ocrDataFields.getStatementOfTruthDateMonth();
+        String year = ocrDataFields.getStatementOfTruthDateYear();
+        if (isEmpty(day) || isEmpty(month) || isEmpty(year)) {
+            caseData.getTransformationAndOcrWarnings().add("Please review statement of truth date for applicant1 in scanned form");
+        }
+        caseData.getPaperFormDetails().setApplicant1SOTSignedOn(
+            deriveStatementOfTruthDate(day, month, year)
+        );
+    }
+
+    private String deriveStatementOfTruthDate(String day, String month, String year) {
+        return Stream.of(day, month, year)
+            .filter(s -> s != null && !s.isEmpty())
+            .collect(Collectors.joining(" "));
+    }
+
+    private void applicant2StatementOfTruth(OcrDataFields ocrDataFields, CaseData caseData) {
+        if (!toBoolean(ocrDataFields.getApplicant2StatementOfTruth())
+            && !toBoolean(ocrDataFields.getApplicant2LegalRepStatementOfTruth())) {
+            caseData.getTransformationAndOcrWarnings().add("Please review statement of truth for applicant2 in scanned form");
+        }
+        if (!toBoolean(ocrDataFields.getApplicant2Signing())
+            && !toBoolean(ocrDataFields.getApplicant2LegalRepSigning())) {
+            caseData.getTransformationAndOcrWarnings().add("Please review statement of truth for applicant2 in scanned form");
+        }
+        if (isEmpty(ocrDataFields.getApplicant2OrLegalRepFullName())) {
+            caseData.getTransformationAndOcrWarnings().add("Please sole or applicant2/legal representative name in scanned form");
+        }
+
+        caseData.getPaperFormDetails().setApplicant2SigningSOT(
+            from(toBoolean(ocrDataFields.getApplicant2Signing()))
+        );
+        caseData.getPaperFormDetails().setApplicant2LegalRepSigningSOT(
+            from(toBoolean(ocrDataFields.getApplicant2LegalRepSigning()))
+        );
+
+        caseData.getPaperFormDetails().setApplicant2LegalRepPosition(ocrDataFields.getApplicant2LegalRepPosition());
+
+        String day = ocrDataFields.getApplicant2StatementOfTruthDateDay();
+        String month = ocrDataFields.getApplicant2StatementOfTruthDateMonth();
+        String year = ocrDataFields.getApplicant2StatementOfTruthDateYear();
+        if (isEmpty(day) || isEmpty(month) || isEmpty(year)) {
+            caseData.getTransformationAndOcrWarnings().add("Please review statement of truth date for applicant2 in scanned form");
+        }
+        caseData.getPaperFormDetails().setApplicant1SOTSignedOn(
```

Comment:
I just noticed that you are setting this both in the applicant 2 SOT method and applicant 1 SOT methods